svo (inker).xml

```diff
@@ -83,7 +83,7 @@
                 <name>Finished inking the tattoo</name>
                 <script>svo.ti_finishedinking()</script>
                 <triggerType>0</triggerType>
-                <conditonLineDelta>0</conditonLineDelta>
+                <conditonLineDelta>1</conditonLineDelta>
```

Comment:
The Delta should be `0` (they should be the same line). As it is now if a line starts with "As the " but is not the ending line for tattooing, it will try to match this line to the regex (can't be avoided) and the next one.